react-native/react/constants/platform.native.desktop.js

```diff
@@ -20,26 +20,43 @@ const envedPathOSX = {
   prod: 'Keybase'
 }
 
+function buildWin32SocketRoot() {
+  let appdata = process.env.APPDATA
+  // Remove leading drive letter e.g. C:
+  if (/^[a-zA-Z]:/.test(appdata)) {
+    appdata = appdata.slice(2)
+  }
+  // Handle runModes, prod has no extension.
+  let extension = ''
+  if (runMode !== 'production') {
+    extension = runMode.charAt(0).toUpperCase() + runMode.substr(1)
+  }
+  let path = `\\\\.\\pipe\\kbservice${appdata}\\Keybase${extension}`
+  return path
+}
+
 function findSocketRoot () {
   const paths = {
     'darwin': `${process.env.HOME}/Library/Caches/${envedPathOSX[runMode]}/`,
-    'linux': runMode === 'prod' ? `${process.env.XDG_RUNTIME_DIR}/keybase/` : `${process.env.XDG_RUNTIME_DIR}/keybase.${runMode}/`
+    'linux': runMode === 'prod' ? `${process.env.XDG_RUNTIME_DIR}/keybase/` : `${process.env.XDG_RUNTIME_DIR}/keybase.${runMode}/`,
```

Comment:
Oh thanks!  But I don't think we want to do this.  We need to duplicate the service's algorithm precisely to find the socket, and this API returns $XDG_CONFIG_HOME instead of $XDG_RUNTIME_DIR on Linux.